bootstrap/lumen.php

```diff
@@ -104,6 +102,7 @@
     'namespace' => 'Ushahidi\App\Http\Controllers',
 ], function ($router) {
     require __DIR__.'/../routes/web.php';
+    require __DIR__.'/../routes/features.php';
```

Comment:
Maybe we should split up routes per entity endpoints and require each one within the `web.php` file?

Reply:
Yeah, splitting per endpoint would be ideal. We'll need a ticket for it,

Reply:
Agree on splitting per endpoints, that'd be nice. We could keep the CheckFeature and separate everything per endpoint , instead of having the routes/feature.php

Reply:
but another option is that everything has their own file , unless feature flagged, I guess? I kind of like the idea of being able to check which flags are still being used easily.